lmp/build.sh

```diff
@@ -130,6 +130,13 @@ if [ -d "${archive}" ] ; then
 	cp ${DEPLOY_DIR_IMAGE}/bbl* ${archive}/other/ || true
 	cp ${DEPLOY_DIR_IMAGE}/fw_* ${archive}/other/ || true
 
+	# Handle user provided extra artifacts
+	if [ ! -z "${EXTRA_ARTIFACTS}" ]; then
+		for extra_file in ${EXTRA_ARTIFACTS}; do
+			cp ${DEPLOY_DIR_IMAGE}/${extra_file} ${archive}/
```

Comment:
Should we publish these additional files to ${archive}/other as done with the files that are not the final image?

Reply:
In this case the user has explicitly asked for the artifact and I was placing them at the top level for visibility.